app/services/stock/routes/routes.js

```diff
@@ -1 +1,12 @@
 const express = require('express');
+const router = express.Router();
+
+const equityController = require('../controllers/controller.js');
+
+router.get('', (req, res, next) => {
```

Comment:
Is this just placeholder until we figure out a name for the endpoint?

Reply:
yes